workers/unity/Packages/io.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -214,6 +217,15 @@ protected ConnectionParameters CreateConnectionParameters(string workerType, ICo
             };
 
             initializer?.Initialize(@params);
+            return @params;
+        }
+
+        protected ConnectionParameters CreateConnectionParameters(string workerId, string workerType,
+            IConnectionParameterInitializer initializer = null)
+        {
+            protocolLog = new ProtocolLogController(workerId);
+            var @params = CreateConnectionParameters(workerType, initializer);
+            @params.Logsinks.Add(protocolLog.LogsinkParameters);
 
             return @params;
         }
```

Comment:
Couldn't this instead be implemented as a `IConnectionParameterInitializer`? Rather than an overload?

Reply:
It could, but the `protocolLogController` would need to be moved to the `GameLogicWorkerConnector` because we need the workerId which isn't really known until after the ConnectionFlow gets initialized.